### PR TITLE
[IMP] mrp_operations_time_control: new requirement

### PR DESCRIPTION
**Pull request by agaldona**
Enable to finish an operation in pause state. The ending date will not be updated.

### CODE REVIEW

File: mrp_operations_time_control/models/operation_time.py [hunk 1]
```diff
@@ -42,8 +42,10 @@ def action_resume(self):
 
     @api.multi
     def action_done(self):
+        not_paused_records = self.filtered(lambda x: x.state != 'pause')
         result = super(MrpProductionWorkcenterLine, self).action_done()
-        self._write_end_date_operation_line()
+        if not_paused_records:
```
Comment (pedrobaeza) [inline mrp_operations_time_control/models/operation_time.py]: You don't need the if.

File: mrp_operations_time_control/views/operation_time_view.xml [hunk 2]
```diff
@@ -36,6 +36,9 @@
                         <field name="operation_time_lines" />
                     </page>
                 </page>
+	            <button name="button_done" position="attributes">
```
Comment (pedrobaeza) [inline mrp_operations_time_control/views/operation_time_view.xml]: Put the same indentation

**Comment by agaldona** [comment]
I've made the changes.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by oihane** [comment]
Looks good to me :+1:

**Comment by mikelarre** [comment]
:+1:

**Pull request merged**